endpoints/manga/controller.py

```diff
@@ -27,45 +29,43 @@
 
 
 class UserMangaResource(Resource):
+    @jwt_required
     def post(self):
         try:
+            current_user = get_jwt_identity()
             manga = request.get_json()
-            db.session.add(UserManga(**manga))
-            db.session.commit()
+            manga['user_id'] = current_user
+            userMangaRepository.add(manga)
             return marshal(manga, manga_list_fields)
 
-        except Exception as error:
-            response.errorResponse(str(error))
-            return response.__dict__
+        except:
+            return {'message': 'Something went wrong', 'timestamp': round(time.time())}, 500
 
+    @jwt_required
     def get(self):
         try:
-            manga = UserManga.query.all()
-            response.successMessage(manga)
+            manga = userMangaRepository.find()
             return marshal(manga, manga_list_fields)
 
-        except Exception as error:
-            response.errorResponse(str(error))
-            return response.__dict__
+        except:
+            return {'message': 'Something went wrong', 'timestamp': round(time.time())}, 500
 
 
 class UserMangaByIdResource(Resource):
-    def get(self, id=None):
+    @jwt_required
+    def get(self, id):
         try:
-            manga = UserManga.query.filter_by(id=id).first()
+            manga = userMangaRepository.findById(id)
```

Comment:
What if the resource doesn't exist? You should probably return 404 in this case. Not an empty resource. I'm guessing this endpoint is /user/<user_id>/manga/<manga_id>